pandas/core/reshape/reshape.py

```diff
@@ -374,15 +374,18 @@ def pivot(self, index=None, columns=None, values=None):
         cols = [columns] if index is None else [index, columns]
         append = index is None
         indexed = self.set_index(cols, append=append)
-        return indexed.unstack(columns)
     else:
-        if index is None:
-            index = self.index
+        index = self.index if index is None else self[index]
+        index = MultiIndex.from_arrays([index, self[columns]])
+        if is_list_like(values):
```

Comment:
I think you need:

``is_list_like(values) and not isinstance(values, tuple)``

Reply:
did you need to check if its a tuple?

Reply:
Yes, I'm trying to figure out why the MultiIndex with a tuple of values test is giving me an error, would be great if you give me a hand!

Reply:
here @jreback: https://travis-ci.org/pandas-dev/pandas/jobs/325099021#L2255